src/components/AnnotationPopover/AnnotationPopover.js

```diff
@@ -46,6 +47,18 @@ class AnnotationPopover extends React.PureComponent<Props> {
         position();
     }
 
+    getHotkeyConfig() {
+        const { onCancel } = this.props;
+        return [
+            new HotkeyRecord({
+                description: 'Close popover',
```

Comment:
Since `onCancel` is passed in and is a noop by default, we don't really know how it's being used. Also, should we still add a hotkey even if `onCancel` isn't defined?